RLBotDiscordBot/calendars.py

```diff
@@ -0,0 +1,156 @@
+import requests
+import datetime
+import json
+import logging
+import sys
+import random as r
+from urllib.parse import urlparse
+import re
+import os
+from config import GOOGLE_API_KEY
+
+import discord
+from discord.ext import commands
+from discord.utils import get
+
+
+async def checkCalendar(message):
+    load_dotenv()
+    api_key = GOOGLE_API_KEY
+    args = message.content.split(" ")
+    can_run = False
+    for i in args:
+        if i == "!tournaments":
+            can_run = True
+    if can_run:
+        months = ["January", "February", "March", "April", "May", "June", "July", "August", "September","October", "November", "December"]
+        today = str(datetime.datetime.now())
+        checkDay = today.split(" ")[0] + "T"
+        checkTime = today.split(" ")[1]
+        checkTime = checkTime.split(":")
+        checkTime[2] = str(int(float(checkTime[2])))
+        checkTime[2] = checkTime[2] + "0Z" if checkTime[2] == "0" else checkTime[2] + "Z"
+        checkTime = ":".join(checkTime)
+        toCheck = checkDay + checkTime
+        r = requests.get(f"https://www.googleapis.com/calendar/v3/calendars/rlbotofficial@gmail.com/events?maxResults=3&timeMin={toCheck}&key={api_key}")
+        jsons = r.json()
+        if len(jsons["items"]) > 0:
+            name = jsons["items"][0]["summary"]
+            start = jsons["items"][0]["start"]["dateTime"]
+            start = start.split("T")
+            date = start[0]
```

Comment:
All of the date parsing can be done much easier with strptime()
https://docs.python.org/3/library/datetime.html#datetime.datetime.strptime
https://docs.python.org/3/library/datetime.html#strftime-strptime-behavior